### DISCUSSION TITLE
Unable to Install Homebrew

### DISCUSSION THREAD

**Discussion by Yash-Shindey** [Homebrew/discussions#4088] [Getting started]
### Output of `brew config`

```shell
(base) yashshindey@Yashs-MacBook-Air ~ % /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
curl: (35) error:140943E8:SSL routines:ssl3_read_bytes:reason(1000)
(base) yashshindey@Yashs-MacBook-Air ~ %
```

### Output of `brew doctor`

```shell
(base) yashshindey@Yashs-MacBook-Air ~ % /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
curl: (35) error:140943E8:SSL routines:ssl3_read_bytes:reason(1000)
(base) yashshindey@Yashs-MacBook-Air ~ %
```

### Description of issue

I am unable to install homebrew on my MacOS Monterey. I keep getting some sort of SSL error and want to solve it. I have previously attempted client server socket programming between two terminals as a school project and wonder if it is related to my SSL error right now. Any Inputs GREATLY APPRECIATED

**Comment by SMillerDev** [comment]
You can try opening the url normally, but we can't help with network issues from your machine to GitHub.

**Reply by Yash-Shindey** [reply]
Thanks for the input, I ended up using a VPN to install Homebrew and it's working fine now
